astropy/utils/tests/test_data.py

```diff
@@ -98,19 +107,20 @@ def test_find_by_hash():
 
 
 # Package data functions
-@pytest.mark.parametrize(('filename'), ['local.dat', 'local.dat.gz', 'local.dat.bz2'])
+@pytest.mark.parametrize(('filename'), ['local.dat', 'local.dat.gz', 'local.dat.bz2', 'local.dat.xz'])
 def test_local_data_obj(filename):
```

Comment:
This test now seems to be leaving open a file handle somewhere--some of the build configurations test that all tests close any files they open.  It's not obvious to me yet where that's happening, but you can try it yourself locally by passing the `--open-files` flag to `./setup.py test`.